From cfb4827326b6743cb732b92580152bcf46647b2c Mon Sep 17 00:00:00 2001
From: Andrew Purtell <apurtell@apache.org>
Date: Fri, 26 Jun 2015 15:04:03 -0700
Subject: [PATCH 1/2] Amend HBASE-13356 HBase should provide an InputFormat
 supporting multiple scans in mapreduce jobs over snapshots (Andrew Mains)

Fix compilation issue for Hadoop 1
---
 .../src/main/java/org/apache/hadoop/hbase/util/ConfigurationUtil.java  | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/util/ConfigurationUtil.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/util/ConfigurationUtil.java
index cd707f3..021e15c 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/util/ConfigurationUtil.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/util/ConfigurationUtil.java
@@ -19,10 +19,11 @@
 package org.apache.hadoop.hbase.util;
 
 import com.google.common.collect.Lists;
+
+import org.apache.commons.lang.StringUtils;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.hbase.classification.InterfaceAudience;
 import org.apache.hadoop.hbase.classification.InterfaceStability;
-import org.apache.hadoop.util.StringUtils;
 
 import java.util.AbstractMap;
 import java.util.Collection;
-- 
2.2.2

